iModelCore/ECPresentation/PublicAPI/ECPresentation/Rules/PresentationRuleSet.h

```diff
@@ -244,6 +244,8 @@ struct PresentationRuleSet : public RefCountedBase, HashableBase
 
         //! Collection of default property category overrides
         DefaultPropertyCategoryOverridesList const& GetDefaultPropertyCategoryOverrides() const {return m_defaultPropertyCategoryOverrides;}
+
+        ECPRESENTATION_EXPORT void CreateIndexes();

```

Comment:
I'd rename this to `AssignRuleIndexes()`. I don't like the term "create" since we're not really creating anything, but assigning. Also, "create indexes" suggests the method should return the created indexes - it's not clear what's really happening with them.

Reply:
renamed